profile_library/govee/H7020/model.json

```diff
@@ -0,0 +1,15 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-02-11T20:22:23.527420",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly PM Mini Gen3",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "master"
+  },
+  "name": "Govee H7020 String lights",
+  "standby_power": 1.8,
```

Comment:
that's some hefty standby usage. Could you double check?

Reply:
I can confirm 1.8 watts using both my testing Shelly 1PM Mini Gen3 and Shelly PM Mini Gen3

Reply:
thanks for verifying